justgage.js

```diff
@@ -996,6 +996,30 @@
     obj, displayVal, color, max, min = null;
   };
 
+  /**
+   * Update Gauge options
+   *
+   * @param {String} option The target option name
+   * @param {Number|String} val The value to be assigned to the option
+   */
+  JustGage.prototype.update = function (option, val) {
+    var obj = this;
+
+    switch (option) {
```

Comment:
If option is not an object option becomes `{[option] : value}`. Here create a `forin` loop to iterate all properties.